ChiDoc/Install_ubuntu_easy.md

```diff
@@ -38,21 +39,27 @@ sudo apt-get install build-essential gfortran
 If you still do not get the appropriate version when running either ``gcc --version``
 or ``gfortran --version`` then there are many online resources which may be able to
 assist you. One issue commonly seen is that the repositories on your system are not
-updated in which case just run ```sudo apt update``` followed by 
+updated in which case just run ```sudo apt update```, possibly followed by 
 ```sudo apt upgrade```.
 
 Now, install the remaining packages needed to build ChiTech and its dependencies:
 
 ```bash
-sudo apt-get install cmake python git zlib1g-dev libx11-dev unzip
-sudo apt-get install libglu1-mesa-dev freeglut3-dev mesa-common-dev
+sudo apt-get install cmake python3 git zlib1g-dev libx11-dev unzip
 ```
+(note the use of python3)
+
 
+
+***TODO: this needs to be fixed @JANV ***
 <u>NOTE</u>: *The recommended version of PETSc requires Python 2.6+; Python 3.x, typically
```

Comment:
We remove this entire line. The newer PETSc versions are fine with python3 only.